setup-old-fashioned

```diff
@@ -98,8 +98,8 @@ def ubuntu_distro_info(options):
 
 
 if __name__ == '__main__':
-    if '18.04' not in DISTRO_VERSION and '16.04' not in DISTRO_VERSION:
-        print("Old-fashioned must be run on Bionic or Xenial Ubuntu.")
+    if '20.04' not in DISTRO_VERSION and '18.04' not in DISTRO_VERSION and '16.04' not in DISTRO_VERSION:
```

Comment:
shouldn't that be `if '20.04' not in DISTRO_VERSION:`  ?

Reply:
> shouldn't that be `if '20.04' not in DISTRO_VERSION:` ?

Sorry it seems the update hadn't been force pushed as expected, I've tried again and it seems to be ok now

And #89 proposed an alternative to not enforce the serie which is an alternative variant of the fix